shenyu-plugin/shenyu-plugin-base/src/main/java/org/apache/shenyu/plugin/base/AbstractShenyuPlugin.java

```diff
@@ -79,58 +79,33 @@ public abstract class AbstractShenyuPlugin implements ShenyuPlugin {
     @Override
     public Mono<Void> execute(final ServerWebExchange exchange, final ShenyuPluginChain chain) {
         initMatchCacheConfig();
-        String pluginName = named();
-        PluginData pluginData = BaseDataCache.getInstance().obtainPluginData(pluginName);
-        if (Objects.nonNull(pluginData) && pluginData.getEnabled()) {
-            final String path = exchange.getRequest().getURI().getPath();
-            SelectorData selectorData = obtainSelectorDataCacheIfEnabled(exchange);
-            if (Objects.isNull(selectorData)) {
-                List<SelectorData> selectors = BaseDataCache.getInstance().obtainSelectorData(pluginName);
-                if (CollectionUtils.isEmpty(selectors)) {
-                    return handleSelectorIfNull(pluginName, exchange, chain);
-                }
-                Pair<Boolean, SelectorData> matchSelectorData = matchSelector(exchange, selectors);
-                selectorData = matchSelectorData.getRight();
-                if (Objects.isNull(selectorData)) {
-                    if (matchCacheConfig.getEnabled() && matchSelectorData.getLeft()) {
-                        selectorData = new SelectorData();
-                        selectorData.setPluginName(named());
-                        cacheSelectorData(path, selectorData);
-                    }
-                    return handleSelectorIfNull(pluginName, exchange, chain);
-                } else {
-                    if (matchCacheConfig.getEnabled() && matchSelectorData.getLeft()) {
-                        cacheSelectorData(path, selectorData);
-                    }
-                }
-            } else {
-                if (StringUtils.isBlank(selectorData.getId())) {
```

Comment:
why remove this，
 if SelectorData not null ，&& selectorData.getId() is null   ，need 
return handleSelectorIfNull(pluginName, exchange, chain);

Reply:
thats not gonna happen,because when cached selectorData.getId() is null ,obtainMatchedSelectorData will return null
![image](https://user-images.githubusercontent.com/42602026/198011389-dd57a69e-3048-4033-9ccf-9cbbb5dbbdf1.png)

![image](https://user-images.githubusercontent.com/42602026/198011099-a5cac872-3653-41ad-b186-8eef1ee282fc.png)